src/main/scala/mesosphere/marathon/state/Container.scala

```diff
@@ -172,23 +170,6 @@ object Container {
         protocol: String = "tcp") {
 
       require(protocol == "tcp" || protocol == "udp", "protocol can only be 'tcp' or 'udp'")
```

Comment:
I added a validator but kept the require … there are various usages of require that we might remove, but I'll leave that for a separate PR.